src/components/user/tour/tour.css

```diff
@@ -0,0 +1,9 @@
+.create-tour-button {
```

Comment:
No need to create separate CSS files. 
Almost everything related to styling should be done by Material UI's default theme. 
For layout purposes, you can use the `Box` component. For more details please refer to the documentation: https://material-ui.com/components/box/